settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'idam-client'
```

Comment:
can we actually just put this in the build.gradle name = 'idam-client' ?
or maybe we should move the group and version here too  so its easy to see the group:artefact:version 😄

Reply:
fine how it is IMO, https://stackoverflow.com/questions/17262856/how-to-set-the-project-name-group-version-plus-source-target-compatibility-in